system-tests/conftest.py

```diff
@@ -7,6 +7,41 @@
 from datetime import datetime
 from time import sleep
 from helpers.timehelpers import unix_time_milliseconds
+from subprocess import Popen
```

Comment:
This whole file could use some improved formatting. Maybe apply the same tool we use for nexus-constructor? (I don't remember the name of it right now)

Reply:
Formatting applied and check added to Jenkins